.github/workflows/release-plz.yaml

```diff
@@ -63,12 +86,53 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.gh_token }}
           CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
 
-      - name: Notify about failure
+      - name: Update ownership
+        shell: 'bash -ex {0}'
+        if: ${{ inputs.org-owner != '' }}
+        run: |
+          cd $(dirname ${{ inputs.manifest_path }})
+          ORG_OWNER=${{ inputs.org-owner }}
+          cargo install cargo-workspaces
```

Comment:
Wouldn't it be better to install it separately (so that e.g. it doesn't randomly fail _after_ publishing?

Reply:
Good catch, it makes sense. Fixed.